tests/queries/0_stateless/02180_group_by_lowcardinality.sql

```diff
@@ -0,0 +1,14 @@
+create database if not exists d_group_by_lowcardinality;
+
+drop table if exists d_group_by_lowcardinality.t_group_by_lowcardinality;
+create table if not exists d_group_by_lowcardinality.t_group_by_lowcardinality(p_date Date, val LowCardinality(Nullable(String))) 
+engine=MergeTree() partition by p_date order by tuple();
+
+insert into d_group_by_lowcardinality.t_group_by_lowcardinality select today() as p_date, toString(number/5) as val from numbers(10000);
+insert into d_group_by_lowcardinality.t_group_by_lowcardinality select today() as p_date, Null as val from numbers(100);
+
+select val, avg(toUInt32(val)) from d_group_by_lowcardinality.t_group_by_lowcardinality group by val limit 10 settings max_threads=1, max_rows_to_group_by=100, group_by_overflow_mode='any' format Null;
```

Comment:
Maybe it worth checking the result?

Reply:
<img width="1106" alt="image" src="https://user-images.githubusercontent.com/11146756/153546573-ff7f03d0-0c46-46bd-9216-a1b7e1236958.png">
The result is possible different in tests.

Reply:
It should be consistent in this case (`max_threads=1` should be enough to make it consistent)

Reply:
got it. thanks